core/src/test/scala/org/apache/spark/scheduler/SparkListenerSuite.scala

```diff
@@ -255,21 +262,19 @@ class SparkListenerSuite extends SparkFunSuite with LocalSparkContext with Match
         taskInfoMetrics.map(_._2.executorDeserializeTime),
         stageInfo + " executorDeserializeTime")
 
-      /* Test is disabled (SEE SPARK-2208)
       if (stageInfo.rddInfos.exists(_.name == d4.name)) {
         checkNonZeroAvg(
           taskInfoMetrics.map(_._2.shuffleReadMetrics.get.fetchWaitTime),
           stageInfo + " fetchWaitTime")
       }
-      */
 
       taskInfoMetrics.foreach { case (taskInfo, taskMetrics) =>
-        taskMetrics.resultSize should be > (0L)
+        taskMetrics.resultSize should be > 0L
         if (stageInfo.rddInfos.exists(info => info.name == d2.name || info.name == d3.name)) {
-          taskMetrics.inputMetrics should not be ('defined)
-          taskMetrics.outputMetrics should not be ('defined)
+          taskMetrics.inputMetrics should not be 'defined
+          taskMetrics.outputMetrics should not be 'defined
           taskMetrics.shuffleWriteMetrics should be ('defined)
-          taskMetrics.shuffleWriteMetrics.get.bytesWritten should be > (0L)
+          taskMetrics.shuffleWriteMetrics.get.bytesWritten should be > 0L
```

Comment:
These changes don't seem to matter, yeah, but they also don't do anything and these lines are now inconsistent. They could just be reverted.

So the test passes but now takes 52 seconds. I wonder if something in the middle is a good compromise. 25ms?